src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/TransformFunctions.kt

```diff
@@ -108,12 +104,9 @@ fun transform(licence: EntityLicence): ModelLicence {
     licenceExpiryDate = licence.licenceExpiryDate,
     topupSupervisionStartDate = licence.topupSupervisionStartDate,
     topupSupervisionExpiryDate = licence.topupSupervisionExpiryDate,
-    comFirstName = licence.comFirstName,
-    comLastName = licence.comLastName,
-    comUsername = licence.comUsername,
-    comStaffId = licence.comStaffId,
-    comEmail = licence.comEmail,
-    comTelephone = licence.comTelephone,
+    comUsername = licence.responsibleCom!!.username,
```

Comment:
Loss of telephone number? Does the initial meeting contact number still get set only in the create-licence journey? So  not related to a com, just whatever contact number they put in during the journey? Probably ok, I think.

Reply:
Initial meeting contact gets stored in a field called `appointment_contact`